test/e2e/kuttl/flagd-disabled/00-assert.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
 commands:
   - command: kubectl wait --for=condition=complete job flagd-query-test -n $NAMESPACE
+collectors:
+  - command: kubectl logs -l job-name=flagd-query-test -n $NAMESPACE
```

Comment:
this is nice, we had a similar ticket in KLT to check this functionality I will link this issue there as an example 🚀  
Not sure whether it is needed here but kuttl also has a possibility to export the test report in a [junit style ](https://github.com/keptn/lifecycle-toolkit/pull/1859)

Reply:
@RealAnna yeah, this is a cool feature to extract error details. Specially when we have to use jobs for validations where GH action can't provide access to error Job

I do not think we need JUnit report though. So I will pass on that.

Reply:
Wow this _is_ cool!